web_src/js/features/repo-diff.ts

```diff
@@ -14,7 +13,7 @@ import {
   showElem,
   animateOnce,
   addDelegatedEventListener,
-  createElementFromHTML,
+  createElementFromHTML, queryElems,
```

Comment:
```suggestion
  createElementFromHTML,
  queryElems,
```

Reply:
Since nobody  need to care about these "imports", I think we could just put them in one line

Reply:
Fine with me too.